test/unit/lib/classes/plugin-manager.test.js

```diff
@@ -1609,4 +1617,32 @@ describe('test/unit/lib/classes/PluginManager.test.js', () => {
       })
     ).to.be.eventually.rejected.and.have.property('code', 'DUPLICATE_PLUGIN_DEFINITION');
   });
+
+  it('should pass through an error when trying to load a plugin with syntax error', async () => {
+    await expect(
+      runServerless({
+        fixture: 'plugin',
+        command: 'print',
+        configExt: {
+          plugins: ['./broken-plugin'],
+        },
+      })
+    ).to.be.eventually.rejectedWith(SyntaxError);
```

Comment:
Testing this seems unrleated to this feature, but it's ok, we can keep it

Reply:
This is just for coverage of the added code.

The 11th line of the following part:

https://github.com/serverless/serverless/blob/a4cd2b5a566275a8e41995559db9c77e0edb7586/lib/utils/import-module.js#L7-L11

Reply:
Ok. so this doesn't have to be _syntax error_. We can simply introduce an initialization error (then no eslint patches are needed)